src/internal/utils/content-header-utils.ts

```diff
@@ -5,3 +5,5 @@ import { getGlobalFlag } from './global-flags';
 export const contentHeaderClassName: string = getGlobalFlag('removeHighContrastHeader')
   ? ''
   : 'awsui-context-content-header';
+
+export const isHighContrastHeaderActive = !!getGlobalFlag('removeHighContrastHeader');
```

Comment:
Can we rename this to `shouldRemoveHighContrastHeader`?

Reply:
Can we also have a single `getGlobalFlag` call?

```
export const isHighContrastHeaderActive = !!getGlobalFlag('removeHighContrastHeader');

export const contentHeaderClassName = isHighContrastHeaderActive ? '' : 'awsui-context-content-header';
```

Reply:
```
export const shouldRemoveHighContrastHeader = !!getGlobalFlag('removeHighContrastHeader');
export const contentHeaderClassName = shouldRemoveHighContrastHeader ? '' : 'awsui-context-content-header';
```